components/pipedrive/actions/update-deal/update-deal.mjs

```diff
@@ -1,116 +1,120 @@
-// legacy_hash_id: a_vgi48J
-import { axios } from "@pipedream/platform";
+import pipedriveApp from "../../pipedrive.app.mjs";
 
 export default {
   key: "pipedrive-update-deal",
   name: "Update Deal",
-  description: "Updates the properties of a deal.",
-  version: "0.1.1",
+  description: "Updates the properties of a deal. See the Pipedrive API docs for Deals [here](https://developers.pipedrive.com/docs/api/v1/#!/Deals)",
+  version: "0.1.2",
   type: "action",
   props: {
-    pipedrive: {
-      type: "app",
-      app: "pipedrive",
-    },
-    companydomain: {
-      type: "string",
-      description: "Your company name as registered in Pipedrive, which becomes part of Pipedrive API base url.",
-    },
-    id: {
+    pipedriveApp,
+    dealId: {
       type: "string",
+      label: "Deal ID",
       description: "ID of the deal",
     },
```

Comment:
`dealId` should be set up in the app file with async options.
https://developers.pipedrive.com/docs/api/v1/Deals#getDeals